api/tests/integration/infrastructure/repositories/schooling-registration-repository_test.js

```diff
@@ -302,6 +302,35 @@ describe('Integration | Infrastructure | Repository | schooling-registration-rep
         schoolingRegistration3B.id,
       ]);
     });
+
+    it('should filter out students registered after August 15, 2020', async () => {
+      // given
+      const organizationId = databaseBuilder.factory.buildOrganization().id;
+
+      const beforeTheDate = new Date('2020-08-14T10:00:00Z');
+      const afterTheDate = new Date('2020-08-16T10:00:00Z');
+      databaseBuilder.factory.buildSchoolingRegistration({ organizationId, updatedAt: beforeTheDate });
+      const earlierSchoolingRegistration = databaseBuilder.factory.buildSchoolingRegistration(
+        { organizationId, updatedAt: afterTheDate },
+      );
+
+      await databaseBuilder.commit();
+
+      // when
+      const schoolingRegistrations = await schoolingRegistrationRepository.findByOrganizationIdAndUpdatedAtOrderByDivision(
+        {
+          organizationId,
+          page: {
+            size: 10,
+            number: 1,
+          },
+          filter: {},
+        },
+      );
+
+      // then
+      expect(_.map(schoolingRegistrations.data, 'id')).to.deep.equal([ earlierSchoolingRegistration.id ]);
+    });
```

Comment:
Est-ce que ca ne serait pas plus explicite?
```suggestion
 expect(schoolingRegistrations.data, 'id').to.have.length(1);
 expect(schoolingRegistrations.data[0].id').to.equal(earlierSchoolingRegistration.id);
    });
```

Reply:
Carrément !